docs/build/exec_examples.py

```diff
@@ -46,34 +67,28 @@ def to_string(value: Any) -> str:
 
 
 class MockPrint:
-    def __init__(self, file: Path):
+    def __init__(self, file: Path) -> None:
```

Comment:
perhaps we should add the script to `lint`, `format` and `mypy` in `Makefile`?

Reply:
Done!